ui/selectors/selectors.js

```diff
@@ -327,6 +327,13 @@ export function getInternalAccountByAddress(state, address) {
   );
 }
 
+export function getMaybeSelectedInternalAccount(state) {
+  // Same as `getSelectedInternalAccount`, but might potentially be `undefined`:
+  // - This might happen during the onboarding
+  const accountId = state.metamask.internalAccounts?.selectedAccount;
+  return accountId && state.metamask.internalAccounts?.accounts[accountId];
+}
```

Comment:
I'm a bit unsure about this selector, lets at least use a nullish coalescing operator to be more explicit about the return value
```suggestion
export function getMaybeSelectedInternalAccount(state) {
  // Same as `getSelectedInternalAccount`, but might potentially be `undefined`:
  // - This might happen during the onboarding
  const accountId = state.metamask.internalAccounts?.selectedAccount;
  return accountId ? state.metamask.internalAccounts?.accounts[accountId] : undefined;
}
```